packages/daemon/test/test-endo.js

```diff
@@ -258,3 +258,77 @@ test.serial('closure state lost by restart', async t => {
     t.is(three, 3);
   }
 });
+
+test.serial('persist import-unsafe0 services and their requests', async t => {
+  const { promise: cancelled } = makePromiseKit();
+  const locator = makeLocator('tmp', 'import-unsafe0');
+
+  await stop(locator).catch(() => {});
+  await reset(locator);
+  await start(locator);
+
+  const inboxFinished = (async () => {
+    const { promise: followerCancelled, reject: cancelFollower } =
+      makePromiseKit();
+    cancelled.catch(cancelFollower);
+    const { getBootstrap } = await makeEndoClient(
+      'client',
+      locator.sockPath,
+      followerCancelled,
+    );
+    const bootstrap = getBootstrap();
+    const worker = await E(bootstrap).makeWorker('userWorker');
+    await E(worker).evaluate(
+      `
+      Far('Answer', {
+        value: () => 42,
+      })
+    `,
+      [],
+      [],
+      'answer',
+    );
+    const iteratorRef = E(bootstrap).followInbox();
+    const { value: message } = await E(iteratorRef).next();
+    const { number } = E.get(message);
+    await E(bootstrap).resolve(await number, 'answer');
+  })();
+
+  const workflowFinished = (async () => {
+    const { getBootstrap } = await makeEndoClient(
+      'client',
+      locator.sockPath,
+      cancelled,
+    );
+    const bootstrap = getBootstrap();
+    const w1 = await E(bootstrap).makeWorker('w1');
+    const servicePath = path.join(dirname, 'test', 'service.js');
```

Comment:
is a `service.js` file supposed to be checked in?

```
~/projects/endo/packages/daemon$ yarn test test/test-endo.js -m '*import-unsafe*'
yarn run v1.22.0
$ ava test/test-endo.js -m '*import-unsafe*'

CapTP client exception: (RemoteError(error:captp:Endo#20001)#1)
  ✘ [fail]: persist import-unsafe0 services and their requests Rejected promise returned by test
RemoteError(error:captp:Endo#20001)#1: Cannot find module '/home/connolly/projects/endo/packages/daemon/test/service.js' imported from /home/connolly/projects/endo/packages/daemon/src/worker.js
  at decodeErrorCommon (packages/marshal/src/marshal.js:279:28)
  at decodeErrorFromCapData (packages/marshal/src/marshal.js:295:14)
  at decodeFromCapData (packages/marshal/src/encodeToCapData.js:412:27)
  at unserialize (packages/marshal/src/marshal.js:352:23)
  at CTP_RETURN (packages/captp/src/captp.js:512:16)
  at dispatch (packages/captp/src/captp.js:576:9)
  at packages/daemon/src/connection.js:28:7

  ─

  persist import-unsafe0 services and their requests

  Rejected promise returned by test. Reason:

  Error {
    message: 'Cannot find module \'/home/connolly/projects/endo/packages/daemon/test/service.js\' imported from /home/connolly/projects/endo/packages/daemon/src/worker.js',
  }

  › decodeErrorCommon (packages/marshal/src/marshal.js:279:28)
  › decodeErrorFromCapData (packages/marshal/src/marshal.js:295:14)
  › decodeFromCapData (packages/marshal/src/encodeToCapData.js:412:27)
  › unserialize (packages/marshal/src/marshal.js:352:23)
  › CTP_RETURN (packages/captp/src/captp.js:512:16)
  › dispatch (packages/captp/src/captp.js:576:9)
  › packages/daemon/src/connection.js:28:7

  ─

  1 test failed
```

I'm running a0968f1f3

Reply:
Yes, the `service.js` test fixture is now checked-in. Thank you!